code/modules/mob/living/carbon/emote.dm

```diff
@@ -359,6 +359,8 @@
 		return insectoid_chitter_sound
 	if(H.gender == FEMALE)
 		return female_scream_sound
+	if(H.species.name == "Tajaran")
```

Comment:
When ordered like this, female catbeasts will scream like human females. 
I assume this is intentional but just to be sure.

Reply:
that was unintentional, but after i thought about it briefly, icouldnt remember any cases of female cats screaming in tom&jerry
so i thought it would be better to keep it, and have it only apply to males

i'll apply it to both genders if the people want it

Reply:
Pull a barry and just pitch shift the screams a few octaves up then use that.

Reply:
I like the pitch shifted female idea.

Reply:
after fucking around for 20 minutes with pitch shifting i'm not satisfied with any of the results
if you can do it in a way they sound good, send me the files or the pitch multiplier
otherwise i'll keep it as is

Reply:
The idea is that they sound terrible, since you just pitch shifted the thing.